src/pinecone_client/vector_db.py

```diff
@@ -0,0 +1,54 @@
+import pinecone
+from langchain.embeddings.openai import OpenAIEmbeddings
+from dotenv import load_dotenv
+import os
+from langchain.vectorstores import Pinecone
+
+# 　環境変数の読み込み
+load_dotenv()
+INDEX_NAME = os.getenv("INDEX_NAME")
+PINECONE_API_KEY = os.getenv("PINECONE_API_KEY")
+PINECONE_ENV = os.getenv("PINECONE_ENV")
```

Comment:
```suggestion
PINECONE_ENV = os.enviorn("PINECONE_ENV")
```
たしかこの書き方にすると型はstrだけになるはずです！
ただこれはどちらでも良いと思います🙇
違いは型だけです！